website/www/site/content/en/blog/go-sdk-release.md

```diff
@@ -0,0 +1,161 @@
+Some concessions are made to keep all SDKs together on the same release cycle.
+
+### Language Compatibility
+
+The SDK will be tested at a minimum [Go Programming Language version of 1.16](https://golang.org/doc/devel/release), and use available language features and standard library packages accordingly.
+To maintain a broad compatibility, the Go SDK will not require the latest major version of Go.
+We expect to follow the 2nd newest supported release of the language, with a possible exception when Go 1.18 is released, in order to begin experimenting with [Go Generics](https://go.dev/blog/generics-proposal) in the SDK.
+Release notes will call out when the minimum version of the language changes.
+
+### Package Compatibility
+
+The primary user packages will avoid changing in backwards incompatible ways for core features.
+This is to be inline with Go's notion of the [`import compatibility rule`](https://research.swtch.com/vgo-import).
+
+> If an old package and a new package have the same import path,
+> the new package must be backwards compatible with the old package.
+
+Exceptions to this policy are around newer, experimental, or in development features and are subject to change.
+Such features will have a doc comment noting the experimental status.
+Major changes will be mentioned in the release notes.
+For example, using `beam.WindowInto` with Triggers is currently experimental and may have the API changed in a future release.
+
+Primary user packages include:
+* The main beam package `github.com/apache/beam/sdks/v2/go/pkg/beam`
+* Sub packages under `.../transforms`, `.../io`, `.../runners`, and `.../testing`.
+
+Generally, packages in the module other than the primary user packages are for framework use and are at risk of changing.
+
+### Known Issues
+
+#### Batteries not included.
+* Current native transforms are undertested.
+* IOs may not be written to scale.
+* Go Direct Runner is incomplete and is not portable, prefer using the Python Portable runner, or Flink.
+  * Doesn't support side input windowing. [BEAM-13075](https://issues.apache.org/jira/browse/BEAM-13075)
+  * Doesn't serialize data, making it unlikely to catch coder issues [BEAM-6372](https://issues.apache.org/jira/browse/BEAM-6372)
+  * Can use other general improvements, and become portable [BEAM-11076](https://issues.apache.org/jira/browse/BEAM-11076)
+* Current Trigger API is under iteration and subject to change [BEAM-3304](https://issues.apache.org/jira/browse/BEAM-3304).
+  * API has a possible breaking change between 2.33.0 and 2.34.0, and may change again.
+* Support of the SDK on services, like Google Cloud Dataflow, remains at the service owner's discretion.
+* Need something?
+  * File a ticket in the [Beam JIRA](https://issues.apache.org/jira/issues/?jql=project%20%3D%20BEAM%20AND%20component%20%3D%20sdk-go) and,
+  * Email the [dev@beam.apache.org](mailto:dev@beam.apache.org?subject=%5BGo%20SDK%20Feature%5D) list!
```

Comment:
Consider having consistent `.` usage.

Not sure how best to insert it when there is a JIRA tag, might be simplest to not have any as I have suggested below:
```suggestion
* Current native transforms are undertested
* IOs may not be written to scale
* Go Direct Runner is incomplete and is not portable, prefer using the Python Portable runner, or Flink
  * Doesn't support side input windowing [BEAM-13075](https://issues.apache.org/jira/browse/BEAM-13075)
  * Doesn't serialize data, making it unlikely to catch coder issues [BEAM-6372](https://issues.apache.org/jira/browse/BEAM-6372)
  * Can use other general improvements, and become portable [BEAM-11076](https://issues.apache.org/jira/browse/BEAM-11076)
* Current Trigger API is under iteration and subject to change [BEAM-3304](https://issues.apache.org/jira/browse/BEAM-3304).
  * API has a possible breaking change between 2.33.0 and 2.34.0, and may change again
* Support of the SDK on services, like Google Cloud Dataflow, remains at the service owner's discretion
* Need something?
  * File a ticket in the [Beam JIRA](https://issues.apache.org/jira/issues/?jql=project%20%3D%20BEAM%20AND%20component%20%3D%20sdk-go) and,
  * Email the [dev@beam.apache.org](mailto:dev@beam.apache.org?subject=%5BGo%20SDK%20Feature%5D) list!
```